repositories/MerchStoreRepository.ts

```diff
@@ -53,6 +53,16 @@ export class MerchItemRepository extends BaseRepository<MerchandiseItemModel> {
     return this.repository.save(item);
   }
 
+  public async updateMerchItemInCollection(collection: string, hidden: boolean): Promise<void> {
```

Comment:
Nit: `updateMerchItemsInCollection` (plural). Also, instead of passing in a boolean that corresponds to some property, we should pass in some `changes` object that can be `{ hidden: true }`, like the other update methods.

Reply:
This is more flexible and also makes it more obvious what the caller is doing.